### ISSUE TITLE
Calling drop on index caused crash

### ISSUE THREAD

**Issue by sirws** [koopjs/koop-socrata#27]
When I call something like:
http://50.18.49.187/koop/socrata/wa/9ubz-5r4b/ + 'drop'...

I get:
Error: spawn rm ENOENT, Error: spawn rm ENOENT, Error: spawn rm ENOENT

This is Koop/Koop-Socrata on windows.

**Comment by chelm** [comment]
@sirws would you mind adding you config file? I just want to see what set so I can match it.

**Comment by sirws** [comment]
@chelm Bear with me... Are you talking about the \koop-sample-app\config\default.json?

**Comment by sirws** [comment]
{
    "server": {
        "port": 1337
    },
    "data_dir": "/usr/local/koop/",
    "db": {
      "conn": "postgres://user:password@localhost/koopdev"
    }
}

**Comment by chelm** [comment]
@sirws Interesting. Dnt see it locally. It might a windows issue. Whats going on is that "/drop" calls the local file system and tries to remove directories of exported data. It could be that on windows it throws an error where on mac/unix it just ignores it.

**Comment by chelm** [comment]
In fact looking at how koop removes dirs might be the issue...

https://github.com/Esri/koop/blob/master/lib/Files.js#L173-L180

That seems pretty "linuxy" to me. There may be some way to make that work in both Windows and Linux/Mac. you could try to commented some of that code out. 

Also I'm curious if you have tried running the koop test suite on windows?

**Comment by sirws** [comment]
How does "data_dir": "/usr/local/koop/" translate to windows?  Since this isn't a *nix os?

Do you think I need to change the data_dir to something more windozy?

**Comment by sirws** [comment]
Hahah!  No, I haven't run the koop test suite.  Is there some good doc on that?

**Comment by chelm** [comment]
@sirws oh yes! you are probably right on making the data_dir more windows freindly! I bet thaat might help. 

To run the tests just: 

```
cd node_modules/koop
npm install --dev 
npm test 
```

**Comment by sirws** [comment]
@chelm Well, looking at the code, it is calling rm which is not a windows command.  Perhaps I will log a bug to handle that on windows properly.

**Comment by chelm** [comment]
right. Im guessing there is a recursive remove dir module that will work on both OS, we could add that as a dep to Koop so that this will work.

**Comment by chelm** [comment]
This might work, but looks like it has issues on windows too: https://github.com/isaacs/rimraf

**Comment by sirws** [comment]
Are there any local files getting created?  I am not seeing any on windows.  Not sure where they would be.

/koop/socrata/wa/9ubz-5r4b/drop undefined {} {}
debug:  0=D:/koop-sample-app/data/files/socrata:https://data.wa.gov:9ubz-5r4b
debug:  0=D:/koop-sample-app/data/tiles/socrata:https://data.wa.gov:9ubz-5r4b
debug:  0=D:/koop-sample-app/data/thumbs/socrata:https://data.wa.gov:9ubz-5r4b

There is nothing the d:\koop-sample-app\data directory.

**Comment by chelm** [comment]
@sirws Files will only get written when an export is created. And actually this raising a new issue about the dir structure. It'd be cleaner if koop contained the file dirs under the item id first. I'll make an issue for that.
